src/api/migrations/20190619073809-create-article.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-unused-vars */
```

Comment:
@truestbyheart rebase with develop. this should already be on develop due to Frank's update article PR